docs/src/SUMMARY.md

```diff
@@ -26,6 +26,7 @@
     - [Five minutes of theory](./apalache/theory.md)
 - [TLC Configuration Files](./apalache/tlc-config.md)
 - [The Snowcat Type Checker](./apalache/typechecker-snowcat.md)
+- [Known Issues](./apalache/known-issues.md)
```

Comment:
I wonder if this should be after "supported features" (at least), but maybe as the very last section?

Reply:
changed the order